addons/website_mass_mailing/static/src/interactions/popup.js

```diff
@@ -0,0 +1,28 @@
+import { patch } from "@web/core/utils/patch";
+import { Popup } from "@website/interactions/popup/popup";
+
+patch(Popup.prototype, {
+    /**
+     * Prevents the (newsletter) popup to be shown if the user is subscribed.
+     *
+     * @override
+     */
+    canShowPopup() {
+        if (
+            this.el.matches(".o_newsletter_popup")
+            && this.el.querySelector("input.js_subscribe_value, input.js_subscribe_email").disabled // js_subscribe_email is kept by compatibility (it was the old name of js_subscribe_value)
```

Comment:
What if selector returns null ?

Reply:
ah, translating jQuery is a pain, that's the kind of things that aren't that obvious :sweat_smile: